galaxy_ng/app/migrations/0017_populate_repos_and_remotes.py

```diff
@@ -31,7 +31,7 @@
         "next_version": 1,
         "remote": {
             "name": "rh-certified",
-            "url": "https://cloud.redhat.com/api/automation-hub/",
+            "url": "https://console.redhat.com/api/automation-hub/",
```

Comment:
This change will ensure this shows up as `console` for a fresh installation, but for existing installations `0017_populate_repos_and_remotes` will have already run so this will not get applied.

To do that we will also need a new migration that updates the existing database entry's `url` field. Note that the end user may have edited this field from its default value.

Perhaps if it starts with `https://cloud.redhat.com/` then update it to start with `https://console.redhat.com/`